modules/@apostrophecms/widget-type/index.js

```diff
@@ -43,6 +43,20 @@
 // 100% of the permanent properties of the widget in this way.**
 // This is needed for the editing experience.
 //
+// ### `neverLoadSelf`
+//
+// If true, this widget's load method will never recursively load
+// another widget of the same type. This option defaults to `true`.
```

Comment:
Maybe "will never recursively load another instance of the same widget type"? The "instance" bit is the important part there.